dataland-dummy-qa-service/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+// dataland-dummy-qa-service
+
+val sonarSources by extra(sourceSets.asMap.values.flatMap { sourceSet -> sourceSet.allSource })
+val jacocoSources by extra(sonarSources)
+val jacocoClasses by extra(
+    sourceSets.asMap.values.flatMap { sourceSet ->
+        sourceSet.output.classesDirs.flatMap {
+            fileTree(it) {
+                exclude("**/openApiClient/**")
+            }.files
+        }
+    },
+)
+val jacocoVersion: String by project
+val openApiGeneratorTimeOutThresholdInSeconds: String by project
+
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.spring")
+    jacoco
+    id("com.gorylenko.gradle-git-properties")
+    id("org.springframework.boot")
+    kotlin("kapt")
+    id("org.jetbrains.kotlin.plugin.jpa")
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("org.springframework.boot:spring-boot-starter-amqp")
+    implementation("org.testng:testng:7.7.0")
+    implementation("junit:junit:4.13.1")
```

Comment:
testng and junit need to be part of testImplementation.
Why junit 4? Don't we always use 5?

Reply:
I would vote for we don't need those two dependencies.

Reply:
Tested and removed.

Reply:
ok